Cargo.toml

```
@@ -26,13 +26,18 @@ bench = false
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-partiql-parser = { features=["serde"], version = "0.3.*" }
-partiql-source-map = "0.3.*"
-partiql-ast = "0.3.*"
-partiql-logical-planner = "0.3.*"
-partiql-logical = "0.3.*"
-partiql-value = "0.3.*"
-partiql-eval = "0.3.*"
+partiql-parser = { features=["serde"], version = "0.4" }
+partiql-catalog = "0.4"
+partiql-source-map = "0.4"
+partiql-ast = "0.4"
+partiql-logical-planner ="0.4"
+partiql-logical = "0.4"
+partiql-value = "0.4"
+partiql-eval = "0.4"
+partiql-extension-ion = "0.4"
+partiql-extension-ion-functions = "~0.4.1"
```

Should this version be "0.4"?

```suggestion
partiql-extension-ion-functions = "0.4."
```

---

No, requires 0.4.1 for this change: https://github.com/partiql/partiql-lang-rust/pull/377